Pipfile

```diff
@@ -4,6 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+azure-storage-blob = "==2.1.0"
```

Comment:
Last version where `BlockBlobService` exists, new version is `BlobServiceClient`